webapi/src/main/scala/org/knora/webapi/routing/RouteUtilV1.scala

```diff
@@ -273,4 +274,32 @@ object RouteUtilV1 {
     } else {
       throw BadRequestException(s"MIME type ${fileMetadataResponse.internalMimeType} not supported in Knora API v1")
     }
+
+  def verifyNumberOfParams[A](
+    params: Seq[A],
+    errorMessage: String,
+    length: Int
+  ): IO[BadRequestException, Option[Nothing]] =
+    ZIO.fail(BadRequestException(errorMessage)).when(params.length != length)
+
+  def toSparqlEncodedString(str: String, errorMsg: String): ZIO[StringFormatter, BadRequestException, IRI] =
```

Comment:
If this is here defined, is the function named the same in SF ` def toSparqlEncodedString(s: String): Option[String]` used at all?